src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/integration_tests/tests/api/ReportTests.kt

```diff
@@ -29,7 +29,7 @@ class ReportTests : IntegrationTest()
     @Test
     fun `runs report`()
     {
-        val response = apiRequestHelper.post("/reports/minimal/run/",
+        val response = apiRequestHelper.post("/reports/minimal-for-running/run/",
```

Comment:
For the most part the integration tests don't alter the orderly state but running reports does. Previously this didn't matter because the `orderly.server` image was using a different orderly directory (the "git" directory). Rather than do anything more complicated around resetting state before each test I've just added this extra report purely for the runner tests which no other tests reference.